api/openapi-spec/openapi.yaml

```diff
@@ -86,10 +93,25 @@ paths:
           description: Forced variation was already set
         '400':
           description: Invalid user id, experiment key, or variation key
+        '401':
+          description: Unauthorized, invalid JWT
         '403':
           $ref: '#/components/responses/Forbidden'
       requestBody:
         $ref: '#/components/requestBodies/OverrideContext'
+  /oauth/token:
```

Comment:
This one requires X-Optimizely-Sdk-Key header as well, that should be documented.